deptofdefense/move.mil
# Add Weight Scales to Locator Map

## Checklist

I have…


## Summary of Changes

This pull request adds weight scales to the seed data and incorporates those locations into the Locator Maps search functionality. To more easily accommodate this cross-model searching, the existing data model was modified to create a polymorphic `Location` model that belongs to "locatable" models (e.g. `TransportationOffice`, `ShippingOffice`, `Installation`).

This PR resolves #85.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-weight-scales`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails server`,
1. load up http://localhost:3000/resources/locator-maps, perform a search, and check out those search results!

## Screenshots

![weight-scales-01](https://user-images.githubusercontent.com/27780860/34264070-a480c678-e63f-11e7-90dc-879d986e9987.png)

![weight-scales-02](https://user-images.githubusercontent.com/27780860/34264146-e6b406ea-e63f-11e7-8674-eb1cd0c3b012.png)
